From 95298656290f1f82258c2c45cf75dd228ddf5281 Mon Sep 17 00:00:00 2001
From: Kevin Wang <Kevin1.Wang@amd.com>
Date: Tue, 9 Jan 2018 14:43:02 +0800
Subject: [PATCH 4629/5855] drm/amdkcl: [4.10] Fix kcl for drm_debugfs_crc
 files

amdgpu_dm_crtc_set_crc_source()
amdgpu_dm_crtc_handle_crc_irq()

if drm version < 4.10, the kernel is not include drm_debugfs_crc.c
so it`s not support it.

Change-Id: I39649784c237a82fa506ba602967365f3421692a
Reviewed-by: Le.Ma <Le.Ma@amd.com>
Signed-off-by: Kevin Wang <Kevin1.Wang@amd.com>

Signed-off-by: Pavan Kumar Ramayanam <pavan.ramayanam@amd.com>
---
 drivers/gpu/drm/amd/display/amdgpu_dm/amdgpu_dm_crc.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/amd/display/amdgpu_dm/amdgpu_dm_crc.c b/drivers/gpu/drm/amd/display/amdgpu_dm/amdgpu_dm_crc.c
index 5768103..2d2a223 100644
--- a/drivers/gpu/drm/amd/display/amdgpu_dm/amdgpu_dm_crc.c
+++ b/drivers/gpu/drm/amd/display/amdgpu_dm/amdgpu_dm_crc.c
@@ -109,5 +109,5 @@ void amdgpu_dm_crtc_handle_crc_irq(struct drm_crtc *crtc)
 		return;
 
 	drm_crtc_add_crc_entry(crtc, true,
-			       drm_crtc_accurate_vblank_count(crtc), crcs);
+			       drm_accurate_vblank_count(crtc), crcs);
 }
-- 
2.7.4

